### DISCUSSION TITLE
「译」Ace，CodeMirror 和 Monaco：Web 代码编辑器的对比

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#114] [Announcements]
https://www.xheldon.com/tech/a-comparison-of-code-editor.html 

 最近看到这篇外文，觉得不错，注意这里对比的是「代码编辑器」，与富文本编辑器还是有些许不同的。

**Comment by SolidZORO** [comment]
预便以 --> 预编译。

**Comment by Xheldon** [comment]
> @SolidZORO
> 预便以 --> 预编译。

感谢，已更正。

**Comment by songispm** [comment]
哈哈哈

**Comment by zhangzhen0513** [comment]
> 热别是如果我们想要创造更加复杂的程序的情况下

`热别` -> `特别`

> 例如，我们从未将代码多线程的功能一直到移动端。

`一直` -> `移植`

> 参与 CodeMirror 使用中，移动用户占了讲了 70%！它比在移动端使用 Ace 更能留住用户。有了 CodeMirror 的可插拔性，很明显，这只是我们在移动端提供更多有价值的事情的开始而已，我们将首先从将在移动端缺失的 PC 端的功能移动到移动端的 CodeMirror 开始。

这句有点不通顺

> CodeMirror 仍然处于测试阶段，该项目有很多微笑的错误，

`微笑` -> `微小`

**感谢分享!!!**

**Comment by Xheldon** [comment]
> @zhangzhen0513

感谢更正，又检查了一遍，现在本文应该没有任何错别字了，谢谢！
